concent_api/verifier/tasks.py

```diff
@@ -132,6 +132,9 @@ def verification_result(
             throw=False,
         )
         return
+    except Exception as e:  # pylint: disable=broad-except
+        logging.warning(f"Exception: {e}, Type {type(e)}")
+        return
```

Comment:
Please remove this overly broad exception handler. I agree with pylint here. If there's a problem, the task should crash. The rule is to crash early and loudly so that we can identify the problems and fix them while still in development and testing. Leave the crash for Celery or Kubernetes to handle. Only catch specific exceptions in specifc situations that you know need to be handled in a special way (like the `DatabaseError` above; though I think that catching all `DatabaseError`s is also a bit too broad there - we only want to handle timeouts not for example errors in the query or in the schema).

You can log it but it's definitely not just a `warning`. But I think the exception will be logged anyway if it escapes from here so it's probably not necessary.

Reply:
I think that the only thing that can go wrong here is `Subtask.DoesNotExists` error.

Reply:
I will remove it.

Reply:
done